spec/shared_examples.rb

```diff
@@ -26,12 +26,19 @@
   its(symbol) { is_expected.to eq(df) }
 end
 
+RSpec.shared_examples 'redis importer' do
```

Comment:
Shared examples that are used by exactly one `_spec.rb` file, SHOULD be in that file. Otherwise it is pretty hard to read.

Also, instead of defining let's, for cases like this it is generally cleaner to pass arguments:
```ruby
it_behaves_like 'redis importer', 
  index: ..., 
  vectors: ...,
  data: ...
```

And also please notice that nothing specific to Redis here! So the context (or even one single matcher!) can be called just "match dataframe unordered" or something like that.

(The above is related to almost _all_ shared contexts in this file)